sympy/assumptions/ask.py

```diff
@@ -8,9 +8,10 @@
 from sympy.core.decorators import deprecated
 from sympy.core.relational import Relational
 from sympy.logic.boolalg import (to_cnf, And, Not, Or, Implies, Equivalent,
-    BooleanFunction, BooleanAtom)
+                                 BooleanFunction, BooleanAtom)
 from sympy.logic.inference import satisfiable
 from sympy.utilities.decorator import memoize_property
+from sympy.assumptions.CNF import CNF, EncodedCNF, Literal
```

Comment:
The module should have a lower case name.

Reply:
Yes please use lowercase names for modules. Uppercase file names can cause issues on systems with case insensitive filesystems (like macOS or Windows).